src/BrandCard/BrandCard.tsx

```diff
@@ -18,7 +18,7 @@ export type BrandCardProps = {
   visualHeight?: string
   /** render image at bottom of card */
   visualBottom?: boolean
-  bgColour: Color
+  bgColor: Color
```

Comment:
have you checked if any of our apps use this component with the `bgColour` prop? I agree that it should be renamed, but it might be worth mentioning in the changelog to avoid ui issues.

Reply:
:eyes: i had a look and its not used at all in any of our repos - ill add a comment to the PR mentioning this tho so semantic-releases will add it to the changelog